frontend/src/app/containers/ExperimentPage.js

```diff
@@ -588,11 +599,23 @@ export class ExperimentDetail extends React.Component {
     return null;
   }
 
+  renderMaximumVersionNotice(title, hasAddon, max_release) {
+    if (hasAddon && !this.maxVersionCheck(max_release)) {
+      return (
+        <div className="upgrade-notice">
+          <div data-l10n-id="versionChangeNotice">This experiment doesn't support your FireFox version, try downloading the stable release.</div>
+          <a onClick={e => this.clickUpgradeNotice(e)} data-l10n-id="versionChangeNoticeLink" href="https://www.mozilla.org/en-US/firefox/" target="_blank">Firefox Stable Release</a>
```

Comment:
Does this need /en-US/ or can we omit that and just let mozilla.org handle locales?